README.md

```diff
@@ -3,14 +3,14 @@
 
 This README is a work in progress.
 
-Full API documentation is available at [docs.robinpowered.com](https://docs.robinpowered.com).
+Full API documentation is available at [docs.robinpowered.com](http://docs.robinpowered.com).
 
 [View the Javadoc](http://robinpowered.github.io/robin-java-sdk/javadoc)
 
 ## Download
 
 This is a pre-release "preview" library and currently only available from the 
-[sonatype `snapshots` repository](https://oss.sonatype.org/content/repositories/snapshots/) and not currently in the 
+[sonatype `snapshots` repository](https://oss.sonatype.org/content/repositories/snapshots/com/robinpowered/robin-java-sdk/) and not currently in the
```

Comment:
This isn't exactly true though. The snapshots repository is the root level one. Perhaps you can add the full link, in addition. So:

```markdown
This is a pre-release "preview" library and currently only available from the  [sonatype `snapshots` repository](https://oss.sonatype.org/content/repositories/snapshots/com/robinpowered/robin-java-sdk/) and not currently in the Maven Central Repository.

The jar is available for direct download [here](https://oss.sonatype.org/content/repositories/snapshots/com/robinpowered/robin-java-sdk/).
```